documents/policies/Competition Requirements.md

```diff
@@ -19,12 +19,15 @@ An official WCA Competition must:
 6. Be held on a single date or on two or more consecutive days.
 7. Have at least one official event.
 8. Have at least one competitor.
+   1. The competition should have at least 12 competitors.
```

Comment:
I think this is a worthless addition. What if it doesn't? We would still recognize the competition, anyway.

Reply:
It's more like: "Aim to hold a competition where at least 12 competitors will participate" than anything else -- as it's a "should", I think this is fine. I'm happy to remove it though if that's the consensus!

Reply:
Indeed, I think "if you hold a competition then you should plan it so you get at least 12 competitors, else you better have a good reason" is still a good idea.

> 3. SHOULD   This word, or the adjective "RECOMMENDED", mean that there
>   may exist valid reasons in particular circumstances to ignore a
>   particular item, but the full implications must be understood and
>   carefully weighed before choosing a different course.
> (https://tools.ietf.org/html/rfc2119)

Reply:
But this is something that some competitions (or organizers or delegates) cannot control, where as typically everything else here is. I don't really see the point in adding the line.

Reply:
I also still prefer its removal. Perhaps write a line in the Crash Course suggesting that competitions should only be planned if you expect to have at least 12 competitors.

Reply:
I'm fine with either keeping it or removing it, happy to go with the majority!

Reply:
I am OK keeping it. Or how about saying that a competition should have a "reasonable amount of competitors"? This would cover cases where there is too few or when people do competitions for 30 when the demand is 200+. I know it's vague, but 12 is not a scientific number either.

Reply:
If we wish to keep a line about a minimum number of competitors, I'd prefer something actionable so it can be evaluated. Example: "If a competition has fewer than 12 registered competitors at the end of the registration period or two weeks before the competition (whichever is earlier), the Delegate must contact the WCA Board." This way competitions with very few competitors can be evaluated to determine if they should still take place. Alternatively, I support removing the line completely.

Reply:
The problem with "reasonable" is that different people will understand it differently, while with a fixed number we won't have that problem.

I'm fine with changing the sentence to something like what Bob mentioned.

Reply:
I think my hold on adding what Bob said is that this is such a rare occurrence and will happen very rarely.
Here are the competitions with less than 12 competitors:
|Comp|competitors|
|--|--
|ChinaFMShenyang2011|3
|ChinaFMZhengzhou2011|3
|CubanOpen2016|6
|AkibaExpoOpen2018|6
|ChinaFMGuangzhou2011|6
|FantabulousMinnesota2015|6
|FantabulousMichigan2015|7
|BalakanQis2019|7
|IdealeNabeulOpen2018|7
|DutchOpen2003|7
|ValgaOpen2013|8
|LastMinuteQualifier2017|8
|TolongDiamMedan2018|8
|EdenVillageWinter2017|8
|Nakhchivan2018|8
|BeijingLongEvents2015|8
|RubikSinFronteras2015|9
|AvalonOpen2013|9
|SantiaguinodeVerano2014|9
|WaikatoAutumnOpen2010|9
|PBQUCLA2017|9
|PuyDeDomeOpen2010|9
|JordanOpen2012|9
|LagomarBast2018|9
|ChalmersOpen2005|9
|BogorOpen2013|9
|JogjaMini2015|10
|ArcticOpen2015|10
|MazateOpen2015|10
|Draughons2006|10
|AnotherBaliCubeDay2015|10
|MITFMC2014|10
|EstonianOpen2014|10
|BergshamraBlind2013|10
|SilencioTotal2018|10
|IdahoOpen2005|10
|GijonOpen2014|10
|CowboysFMCParty2017|10
|ChennaiMindOverMatter2018|11
|TartuOpen2016|11
|StadiFMC2018|11
|PleaseBeQuietWPI2017|11
|HeimaeyOpen2018|11
|OSUBlindWeekend2018|11
|JCubeJammu2017|11
|KotkaOpen2011|11
|MiniMinnesotaFall2017|11
|PleaseBeQuietBelgium2018|11
|Svekub2005|11
|IgnusOpen2016|11
|PleaseBeQuietReno2017|11
|SwisscubingFMC2018|11
|MalagaUIS2018|11

Would the board have denied any of these competitions from happening?

Reply:
Probably not. All the more reason why we should get rid of this line and just add it to the crash course instead.

Reply:
Ok, I'm removing the line from the new version I'm working on.